src/test/java/org/sagebionetworks/bridge/BridgeUtilsTest.java

```diff
@@ -1033,6 +1040,34 @@ public void getIntegerOrDefaultThrowsBadRequest() {
         BridgeUtils.getIntegerOrDefault("asdf", null);
     }
     
+    @Test
+    public void convertParsingErrorMismatchedInputException() {
+        MismatchedInputException mie = Mockito.mock(MismatchedInputException.class);
+        Reference ref1 = new Reference(null, "password");
+        Reference ref2 = new Reference(null, "phone");
+        List<Reference> list = ImmutableList.of(ref1, ref2);
+        Mockito.when(mie.getPath()).thenReturn(list);
+        
+        InvalidEntityException e = BridgeUtils.convertParsingError(mie);
+        assertEquals(e.getMessage(), "Error parsing JSON in request body fields: password, phone");
+    }
```

Comment:
Let's take this one step further: Instead of mocking the exception and hoping we got the format right, let's make some malformed JSON and parse the JSON so that we have a real exception to work with.

Reply:
Now throwing a real exception. I also widened the test to the parent exception that includes the getPath() method (although I haven't been able to trigger the more obscure exceptions, like UnresolvedForwardReference, I worry they might also leak information). And added a comma to the error message because rereading it... it was confusing.